freqtrade/freqai/freqai_interface.py

```diff
@@ -703,6 +703,7 @@ def _set_train_queue(self):
         current_pairlist = self.config.get("exchange", {}).get("pair_whitelist")
         if not self.dd.pair_dict:
             logger.info('Set fresh train queue from whitelist.')
+            logger.info(f'Queue: {current_pairlist}')
```

Comment:
Can we combine this message (same below).

FreqAI already creates WAY too many logmessages - so we should aim to combine them where possible (having 2 messages here will provide 0 benefit).